GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -120,47 +149,214 @@ namespace gdjs {
-            object.getDrawableY() + object.getCenterY(),
-            3
-          );
+          // Draw points
+          // Center point of the object, with camera movement
+          const centerPointX =
+            object.getDrawableX() + object.getCenterX() - cameraTopLeftX;
+          const centerPointY =
+            object.getDrawableY() + object.getCenterY() - cameraTopLeftY;
+
+          // Draw Center point
+          debugDraw.fill.alpha = 0.3;
+          debugDraw.line.color = 0xffff00;
+          debugDraw.fill.color = 0xffff00;
+          debugDraw.drawCircle(centerPointX, centerPointY, 3);
+
+          if (showPointsNames) {
+            if (!renderedObjectTextPoints.points['center']) {
+              renderedObjectTextPoints.points['center'] = new PIXI.Text(
+                'Center',
+                {
+                  fill: '#ffff00',
+                  fontSize: 12,
+                }
+              );
+            }
+
+            renderedObjectTextPoints.points['center'].position.set(
+              centerPointX,
+              centerPointY
+            );
+
+            this._pixiContainer.addChild(
+              renderedObjectTextPoints.points['center']
+            );
+          }
+
+          // Draw Origin point
+          // Origin point of the object, with camera movement
+          // For Sprite objects get the position of the origin point.
+          // For others objects that doesn't have origin point the position of the rendered object is used.
+          let originPoint = [
+            object.getDrawableX() - cameraTopLeftX,
+            object.getDrawableY() - cameraTopLeftY,
+          ];
+          if (object instanceof gdjs.SpriteRuntimeObject) {
+            originPoint = object.getPointPosition('origin');
+            originPoint[0] = originPoint[0] - cameraTopLeftX;
+            originPoint[1] = originPoint[1] - cameraTopLeftY;
+          }
+
+          debugDraw.line.color = 0xff0000;
+          debugDraw.fill.color = 0xff0000;
+          debugDraw.drawCircle(originPoint[0], originPoint[1], 3);
+
+          if (showPointsNames) {
+            if (!renderedObjectTextPoints.points['origin']) {
+              renderedObjectTextPoints.points['origin'] = new PIXI.Text(
+                'Origin',
+                {
+                  fill: '#ff0000',
+                  fontSize: 12,
+                }
+              );
+            }
+            renderedObjectTextPoints.points['origin'].position.set(
+              originPoint[0],
+              originPoint[1]
+            );
+
+            // Mark the text points as rendered
+            renderedObjectTextPoints.wasRendered = true;
+            this._pixiContainer.addChild(
+              renderedObjectTextPoints.points['origin']
+            );
+          }
+
+          // Draw custom point
+          if (showCustomPoints && object instanceof gdjs.SpriteRuntimeObject) {
+            if (!object?._animationFrame?.points) continue;
```

Comment:
Don't use ?., it seems it's not translated by esbuild and the tests are failing because of it I think:
![image](https://user-images.githubusercontent.com/1280130/112763147-1feb2700-8ffb-11eb-91d2-99cd21466c51.png)

More generally, using `?.` can almost always be replaced by a simple check and is often a way to say "huh I don't really know if I have something valid or not so I'll just put a ?" ;) 
Here you are sure you have an object. You're also sure that you have a gdjs.SpriteRuntimeObject. So the _animationFrame is either null, or not.
If it's not null, you're sure you have points. So you just need:
`if (!object._animationFrame) continue;`